OpenRA.Mods.Common/Lint/CheckTranslationReference.cs

```diff
@@ -135,6 +136,13 @@ void ILintPass.Run(Action<string> emitError, Action<string> emitWarning, ModData
 				}
 			}
 
+			var chromeFields = new[] { "Text", "TooltipText", "TooltipDesc" };
+			foreach (var filename in modData.Manifest.ChromeLayout)
+			{
+				var nodes = MiniYaml.FromStream(modData.DefaultFileSystem.Open(filename));
+				CheckChrome(nodes, chromeFields, translation, language, emitError);
+			}
+
```

Comment:
I now see the discussion was solely contained in discord. I have mentioned it multiple times that this should be using reflection instead, I don't see next release without the proper linting. A good example of how to do this is the chrome hotkey linting

Reply:
Is it worth noting that as a follow-up in the tracking issue #10475? Getting this large batch of translations in to reduce the rebase toil would be a win.

Reply:
I'm waiting on #21003 to be merged first as I want less conflicts between prep and bleed